### PR TITLE
B 20016 int

### PR DESCRIPTION
**Pull request by deandreJones**
## [B-20016](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A949852)
 **GO 1.22.2 UPGRADE**
 
 https://dp3.atlassian.net/wiki/spaces/MT/pages/1931673649/How+to+Upgrade+Go+Version#3.-Update-transcom%2Fmymove-Repo

## Summary
Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author
-currently live in DEMO environment
These are to be checked by the author.

- [x] Tested in the DEMO environment (for changes to containers, app startup, or connection to data stores)
- [ ] Have the Agility acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

- [ ] Has the branch been pulled in and checked out?
- [ ] Have the BL acceptance criteria been met for this change?
- [ ] Was the CircleCI build successful?
- [ ] Has the code been reviewed from a standards and best practices point of view?

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access any app
2. Login as any user
3. make sure app doesn't break

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/getting-started/development/logging).
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/backend/guides/golang-guide#querying-the-database-safely) have been satisfied.

### CODE REVIEW

**Review by josiahzimmerman-caci — APPROVED** [review]

File: .golangci.yml [hunk 1]
```diff
@@ -32,32 +32,29 @@ linters-settings:
...
-    check-shadowing: true
 linters:
   enable:
     - errcheck
     - gci
     - gofmt
     - goimports
     - gomodguard
-    - gosec
+   # - gosec
```
Comment (danieljordan-caci) [inline .golangci.yml:42]: You want these commented out?

Comment (deandreJones) [reply .golangci.yml:42]: I was actually going through those - uncommenting- has a LOT of new findings I need to go through

Comment (danieljordan-caci) [reply .golangci.yml:42]: I'll approve but that was the only weird thing that stood out to me. Go forth, my brave man!

**Review by danieljordan-caci — APPROVED** [review]
LGTM?

**Pull request merged**